docs/installation.rst

```diff
@@ -4,29 +4,34 @@ Installation
 Basic installation
 ------------------
 
-.. warning::
+Start by `installing a suitable version of JAX and jaxlib <https://github.com/google/jax#installation>`_. If you don't plan on using ``mpi4jax`` on GPU, the following will do:
+
+.. code:: bash
+
+   $ pip install jax jaxlib
 
-   Much of the functionality we need has recently been added to JAX, which itself is changing frequently. Therefore, ``mpi4jax`` has somewhat strict requirements on the supported versions of JAX and jaxlib.
+.. note::
+
+   Much of the functionality we need has recently been added to JAX, which itself is changing frequently. Therefore, ``mpi4jax`` has somewhat strict requirements on the supported versions of JAX and jaxlib. Be prepared to upgrade!
 
-We recommend that you use ``pip`` to install mpi4jax, however a distribution is also available on ``conda``:
+We recommend that you use ``pip`` to install mpi4jax (but a distribution is also available via ``conda``):
```

Comment:
```suggestion
We recommend that you use ``pip`` to install mpi4jax (but a distribution is also available via ``conda`` which will work if MPI, mpi4py and mpi4jax are all installed through conda ):
```

Reply:
How is this? I think it is a common best practice that you shouldn't mix conda and pip. Conda users hopefully already know this.